electricitymap/contrib/lib/models/event_lists.py

```diff
@@ -279,6 +279,35 @@ def update_production_breakdowns(
 
         return production_breakdowns
 
+    @staticmethod
+    def filter_only_zero_production(
+        breakdowns: "ProductionBreakdownList",
+    ) -> "ProductionBreakdownList":
+        """A method to filter out production breakdowns with a total production of 0 MW."""
+        events = ProductionBreakdownList(breakdowns.logger)
+        for event in breakdowns.events:
+            total_production = sum(
+                {
+                    k: v for k, v in event.production.dict().items() if v is not None
+                }.values()
+            )
```

Comment:
Is it not possible to call `all()` directly and assert if they are `None or 0` instead?

I think that would be simpler (and faster as it can exit early) if possible but I'm not sure if we have set up the internal methods in the production mix for that.